docs/current-001-node_bootstrap.md

```diff
@@ -113,42 +164,158 @@ ExecStart=/usr/bin/kubelet \
   --network-plugin=cni \
   --cni-bin-dir=/opt/cni/bin \
   --cni-conf-dir=/etc/cni/net.d \
-  --pod-infra-container-image=${AWS_EKS_ECR_ACCOUNT}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/eks/pause-amd64:3.1 \
-  --kubeconfig=/etc/eksctl/kubeconfig.yaml \
-  --config=/etc/eksctl/kubelet.yaml
-```
+  --pod-infra-container-image=${AWS_EKS_ECR_ACCOUNT}.dkr.ecr.${AWS_DEFAULT_REGION}.${AWS_SERVICES_DOMAIN}/eks/pause:3.1-eksbuild.1 \
+  --kubeconfig=<a href="#kubeconfig">/etc/eksctl/kubeconfig.yaml</a> \
```

Comment:
some HTML tags snuck in here

Reply:
That's on purpose! :) I thought adding that to navigate through the different files might be helpful to understand. What do you think?

Reply:
oh cool I didn't realize it works inside \`\`!

Reply:
They only work inside `<pre></pre>` tags, but that makes the block look the same in GH (not in the website though).